wolox-books/src/config/helpers.js

```diff
@@ -0,0 +1,5 @@
+import { helpers } from 'vuelidate/lib/validators'
+
+const validatePassword = helpers.regex('validatePassword', /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]$/)
+
+export default validatePassword
```

Comment:
Instead of `export default`, export the `validatePassword` without being the default import

`export const validatePassword = ...`